### PR TITLE
[4.0] remove <php 7 code

### PR DESCRIPTION
**Pull request by brianteeman**
This pr removes checks fromm the joomla_update code for < 7.0 php functionality. No point in the code being there as it will only ever be run on a site with > 7.0 php

### CODE REVIEW

**Review by mbabker — APPROVED** [review]

File: installation/src/Model/ChecksModel.php [hunk 1]
```diff
@@ -188,13 +188,6 @@ public function getPhpSettings()
 		$setting->recommended = true;
 		$settings[] = $setting;
 
-		// Check for magic quotes runtimes.
```
Comment (dgrammatiko) [inline installation/src/Model/ChecksModel.php:191]: Checks are good to stay as a safety net. IMHO it's better to check over known bad patterns and raise an error than expecting users to fulfil the prerequisites, hidden in some doc page.

IFRC this code will raise the first page (requirements not met) so good to stay. That's for the clean install, I have no clue what's the procedure on the update

Comment (mbabker) [reply installation/src/Model/ChecksModel.php:191]: Magic quotes runtime INI config is gone as of PHP 5.4 and dependent functions in PHP 7.0.  This check literally has zero value in Joomla 4.

Comment (brianteeman) [reply installation/src/Model/ChecksModel.php:191]: There is zero point in having a check that is only valid for < php 7.0 in software that can only run on > php 7.0

Inception

Comment (Quy) [reply installation/src/Model/ChecksModel.php:191]: This too: https://github.com/joomla/joomla-cms/pull/20555/files#diff-f67bf23c32006b9bfac846211cfdf556R170

Comment (wilsonge) [reply installation/src/Model/ChecksModel.php:191]: Yup looks like safe mode was removed in 5.4 (http://php.net/manual/en/features.safe-mode.php) - that can go too

Comment (brianteeman) [reply installation/src/Model/ChecksModel.php:191]: done

File: installation/language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -197,7 +197,6 @@ INSTL_PARSE_INI_FILE_AVAILABLE="INI Parser Support"
 INSTL_PHP_VERSION="PHP Version"
 INSTL_PHP_VERSION_NEWER="PHP Version >= %s"
 INSTL_REGISTER_GLOBALS="Register Globals Off"
```
Comment (Quy) [inline installation/language/en-GB/en-GB.ini]: Remove

File: installation/language/en-US/en-US.ini [hunk 3]
```diff
@@ -196,7 +196,6 @@ INSTL_PARSE_INI_FILE_AVAILABLE="INI Parser Support"
 INSTL_PHP_VERSION="PHP Version"
 INSTL_PHP_VERSION_NEWER="PHP Version >= %s"
 INSTL_REGISTER_GLOBALS="Register Globals Off"
```
Comment (Quy) [inline installation/language/en-US/en-US.ini]: Remove

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6294919e9957feb58aae81ffab9e644f4f7442c6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20555">issues.joomla.org/tracker/joomla-cms/20555</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**